etc/migration/disable_users_from_tenancy/1.disable-users-by-tenancy.js

```diff
@@ -82,11 +82,11 @@ OAE.init(config, function (err) {
 
   DisableUsersMigration.doMigration(tenantAlias, function(err, users) {
```

Comment:
I'm going to run `git checkout HEAD~1 etc/migration/disable_users_from_tenancy/1.disable-users-by-tenancy.js` and hopefully "go back in time".